structure_tests/licenses_test.go

```diff
@@ -0,0 +1,66 @@
+package structure_tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+type problem struct {
+	name  string
+	issue string
+}
+
+func TestImageLicenses(t *testing.T) {
+	whitelist := [0]string{}
```

Comment:
Could you move these up to package level variables?

var (
whitelist = []string{}
blacklist = []string{...}
)

with comments explaining each one.